### ISSUE TITLE
Wrong tab highlighted when opening Entity settings card

### ISSUE THREAD

**Issue by kpine** [home-assistant/frontend#12523]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

When opening the Entity settings, the "Settings" tab will be highlighted even if the "Related" tab is focused. This occurs after selecting "Related", closing the card, and re-opening it.

### Describe the behavior you expected

The visible tab should be highlighted, as if it was clicked on and selected.

### Steps to reproduce the issue

1. Open the Entity settings card for any entity (i.e. from the Device page)
2. Select the "Related" tab
4. Close the card
5. Open the Entity settings card again
6. Notice that the "Related" tab is still selected and visible, but the "Settings" tab is highlighted

### What version of Home Assistant Core has the issue?

core-2022.4.7

### In which browser are you experiencing the issue with?

Google Chrome 101.0.4951.41

### Which operating system are you using to run this browser?

Windows 10

### Additional information

Here's an example screenshot after selecting the "Related" tab, and before closing the card (Step 2):
![image](https://user-images.githubusercontent.com/791794/166093960-3e7b3aab-2c80-43d3-a4d0-706b69131648.png)

Here's an example screenshot after immediately opening the Entity settings care after closing it (Step 6):
![image](https://user-images.githubusercontent.com/791794/166094027-8c4e924c-696a-42e4-802d-e83d5d7f17f7.png)

Also reproduced with Firefox 99.0.1.

Also reproduced in 2022.5.0b4 with frontend 20220429.0.

**Comment by steverep** [comment]
I should be able to fix this

**Comment by steverep** [comment]
This and other dialogs with tab bars are set to give initial focus to the first tab, which is not necessarily the active tab (although in most cases it is).

After investigating and jogging my memory, this is a limitation of the `mwc-tab-bar` component.  I filed issue material-components/material-web#3396 to hopefully get that added and then this would be an easy fix.

**Comment by kpine** [comment]
Still an issue.

**Comment by steverep** [comment]
Still an issue

**Comment by steverep** [comment]
You're right, but the problem could easily pop up again because of the dependency issue.  The focused tab and selected tab are not programmatically synced.  We're just lucky they are the same now.

Let's keep this open until it's not fixed by accident.
